op-node/p2p/signer.go

```diff
@@ -30,7 +30,7 @@ func SigningHash(domain [32]byte, chainID *big.Int, payloadBytes []byte) (common
 	}
 	chainID.FillBytes(msgInput[32:64])
 	// payload_hash: third 32 bytes, hash of encoded payload
-	copy(msgInput[32:], crypto.Keccak256(payloadBytes))
```

Comment:
Fix looks good. But adding this comment to resolve to ensure we can ACK that this is getting merged, since it's a breaking change. As soon as this is deployed to the sequencer, replica nodes will stop following the unsafe chain until they update as well. All nodes will stay in sync with the L1 data though, so I think it's fine.